hack/addons/update-calico-manifests.sh

```diff
@@ -13,17 +13,15 @@ if [ -z "${CALICO_VERSION:-}" ]; then
   exit 1
 fi
 
-CALICO_CNI_ASSETS_DIR="$(mktemp -d -p "${TMPDIR:-/tmp}")"
-readonly CALICO_CNI_ASSETS_DIR
-trap 'rm -rf ${CALICO_CNI_ASSETS_DIR}' EXIT
+ASSETS_DIR="$(mktemp -d -p "${TMPDIR:-/tmp}")"
+readonly ASSETS_DIR
+trap 'rm -rf ${ASSETS_DIR}' EXIT
 
-curl -fsSL "https://raw.githubusercontent.com/projectcalico/calico/${CALICO_VERSION}/manifests/tigera-operator.yaml" \
-  -o "${CALICO_CNI_ASSETS_DIR}/tigera-operator.yaml"
+readonly FILE_NAME="tigera-operator.yaml"
 
-readonly KUSTOMIZATION_DIR=${SCRIPT_DIR}/kustomize/tigera-operator
-cp -r "${KUSTOMIZATION_DIR}"/* "${CALICO_CNI_ASSETS_DIR}"
-kustomize --load-restrictor=LoadRestrictionsNone build "${CALICO_CNI_ASSETS_DIR}" \
-  -o "${CALICO_CNI_ASSETS_DIR}/kustomized.yaml"
+readonly KUSTOMIZE_BASE_DIR=${SCRIPT_DIR}/kustomize/tigera-operator
```

Comment:
nit: to be consistent with NFD
```suggestion
readonly KUSTOMIZE_BASE_DIR="${SCRIPT_DIR}/kustomize/tigera-operator/"
```